packages/coinjoin/src/client/round/outputRegistration.ts

```diff
@@ -201,12 +201,14 @@ export const outputRegistration = async (
         // collect all used addresses
         const usedAddresses: AccountAddress[] = [];
         // try to register outputs for each account (each input in account group)
+        round.setSessionPhase(SessionPhase.RegisteringOutputs);
```

Comment:
maybe we can add some more details/progress like: "registering X of Y" output?

Reply:
that would require adjustments to the whole flow, I'd ask Hynek first and do in in another PR. He mentioned even the subject of this PR might be too much, so not sure he would want more details :(

Reply:
Yes, skip that detail for now..